tests/Sample/ListChannelsRoute.php

```diff
@@ -0,0 +1,30 @@
+<?php declare(strict_types = 1);
+
+namespace BrandEmbassyTest\Slim\Sample;
+
+use BrandEmbassy\Slim\Request\RequestInterface;
+use BrandEmbassy\Slim\Response\ResponseInterface;
+use BrandEmbassy\Slim\Route\Route;
+
+final class ListChannelsRoute implements Route
```

Comment:
SampleListChannelsRoute?

Reply:
We don't use that convention here:
![image](https://user-images.githubusercontent.com/10008612/152528097-316c01d2-e829-41c9-875c-20f760786f2b.png)